Makefile

```diff
@@ -664,7 +664,12 @@ unf: $(ROM) $(LOADER_EXEC)
 libultra: $(BUILD_DIR)/libultra.a
 
 patch: $(ROM)
+ifeq ($(shell uname), Darwin)
+  $(error "The 'make patch' command is not supported on macOS.")
```

Comment:
tab moment

Reply:
![anguish](https://www.looper.com/img/gallery/the-scene-that-made-fans-fall-in-love-with-breaking-bad/intro-1650379890.webp)